moped-database/seeds/1602292389297_initial_seed_staging.sql

```diff
@@ -132,7 +132,7 @@ INSERT INTO public.moped_proj_features (feature_id, feature, project_component_i
 
 INSERT INTO public.moped_proj_funding (proj_funding_id, project_id, date_added, added_by, funding_source_id, funding_program_id, funding_amount, fund_dept_unit, funding_description, funding_status_id, fund, dept_unit, is_deleted) VALUES (2, 227, '2022-11-12 18:12:47.295511+00', 10, 4, 7, 0, '', 'cha ching', 2, '{"fund_id": "8400", "fund_name": "GCP-TPSD GENERAL IMPROVEMENTS"}', '{"dept": "2507", "unit": "9024", "dept_id": "180775091", "dept_unit_id": "359318059", "unit_long_name": "Pay Station Parking Meter Replacement", "unit_short_name": "8400-2507", "dept_unit_status": "Active"}', false);
 INSERT INTO public.moped_proj_funding (proj_funding_id, project_id, date_added, added_by, funding_source_id, funding_program_id, funding_amount, fund_dept_unit, funding_description, funding_status_id, fund, dept_unit, is_deleted) VALUES (4, 227, '2022-11-12 18:12:47.295689+00', 10, 9, 11, 0, '', '', 1, '{"fund_id": "4730", "fund_name": "PARKING CIP"}', '{"dept": "2507", "unit": "9040", "dept_id": "180775091", "dept_unit_id": "207413510", "unit_long_name": "Parking Initiatives", "unit_short_name": "8400-2507", "dept_unit_status": "Active"}', false);
-INSERT INTO public.moped_proj_funding (proj_funding_id, project_id, date_added, added_by, funding_source_id, funding_program_id, funding_amount, fund_dept_unit, funding_description, funding_status_id, fund, dept_unit, is_deleted) VALUES (3, 227, '2022-11-12 18:12:47.295683+00', 10, NULL, NULL, NULL, NULL, NULL, 2, '{"fund_id": "4730", "fund_name": "PARKING CIP"}', '{"dept": "2507", "unit": "9042", "dept_id": "180775091", "dept_unit_id": "153705858", "unit_long_name": "New & Repair Parking Stations", "unit_short_name": "4730-2507", "dept_unit_status": "Active"}', true);
+-- INSERT INTO public.moped_proj_funding (proj_funding_id, project_id, date_added, added_by, funding_source_id, funding_program_id, funding_amount, fund_dept_unit, funding_description, funding_status_id, fund, dept_unit, is_deleted) VALUES (3, 227, '2022-11-12 18:12:47.295683+00', 10, NULL, NULL, NULL, NULL, NULL, 2, '{"fund_id": "4730", "fund_name": "PARKING CIP"}', '{"dept": "2507", "unit": "9042", "dept_id": "180775091", "dept_unit_id": "153705858", "unit_long_name": "New & Repair Parking Stations", "unit_short_name": "4730-2507", "dept_unit_status": "Active"}', true);
```

Comment:
?

Reply:
this was an entry where is_deleted is true, so it was being added to the activity log as a change. ill change it to is_deleted being false.